api/extension-guides/chat-tutorial.md

```diff
@@ -9,14 +9,236 @@ MetaDescription: Tutorial that walks you through creating a GitHub Copilot chat
 
 ## Prerequisites
+To get started, you will need the following:
+- Visual Studio Code (VS Code) installed.
+- The GitHub Copilot Chat extension installed, with an active GitHub Copilot subscription.
+- Node.js installed.
 
-## Step 1:
+## Step 1: Set up your project
+First, you will generate the extension project using Yeoman and the VS Code Extension Generator.
 
-## Step 2:
+```bash
+npx --package yo --package generator-code -- yo code
+```
 
+Select the following options to complete the setup:
+
+```bash
+# ? What type of extension do you want to create? New Extension (TypeScript)
+# ? What's the name of your extension? Code Tutor
+
+### Press <Enter> to choose default for all options below ###
+
+# ? What's the identifier of your extension? code-tutor
+# ? What's the description of your extension? LEAVE BLANK
+# ? Initialize a git repository? Yes
+# ? Bundle the source code with webpack? No
+# ? Which package manager to use? npm
+
+# ? Do you want to open the new folder with Visual Studio Code? Open with `code`
+```
+
+Once your extension project is generated, there are two files you will be working in: `extension.ts` and `package.json`, which you can learn more about in the [Extension Anatomy docs](/api/get-started/extension-anatomy#extension-file-structure).
+
+Delete the auto-generated code in the `extension.ts` `activate()` method. This is where you will place our logic for our chat participant.
+
+## Step 2: Register a Chat participant
+In the `package.json` file, replace the auto-generated `contributes` section with the following:
+
+```json
+"contributes":{
+	"chatParticipants": [
+		{
+			"id": "chat-sample.code-tutor",
+			"fullName": "Code Tutor",
+			"name": "tutor",
+			"description": "What can I teach you?",
+			"isSticky": true
+		}
+	]
+}
+```
+
+This code registers a chat participant with the following attributes:
+- Unique ID `chat-sample.code-tutor`, which will be referenced in the code
+- Full name `Code Tutor`, which will be shown in the title area of a response from your participant
+- Name `tutor`, which will be used to reference the chat participant as `@tutor` in the Chat view
+- Description "What can I teach you?", which will be shown in the chat input field as a placeholder text
+
+Finally, setting `isSticky: true` will automatically prepend the participant name in the chat input field after the user has started interacting with the participant.
+
+## Step 3: Craft the prompt and select the model
+Now that the participant is registered, you can start implementing the logic for the code tutor. In the `extension.ts` file, you will define a prompt and select the model for the requests.
+
+Crafting a good prompt is the key to getting the best response from your participant. Check out [this video](https://youtu.be/H3M95i4iS5c) for tips on prompt engineering.
```

Comment:
I am not sure if this video is actually related to what extension authors might need. Since Burke is giving advice on how to get the best out of copilot chat.
And this is more about creating system prompts and making sure LLM behavies well for your chat participant use case. Linking to https://platform.openai.com/docs/guides/prompt-engineering might be better